Cargo.toml

```diff
@@ -11,9 +11,8 @@ An implementation of SSL streams for Tokio backed by OpenSSL
 
 [dependencies]
 futures = "0.3"
-openssl = "0.10.32"
+openssl = "0.10.34"
```

Comment:
What is the purpose of this bump?

Reply:
I thought making sure the deps are up to date would make sense if we are messing with them anyways. Should i revert this?

Reply:
This is just a lower bound on the version Cargo will pick. We need some specific API from 0.10.32 which is why it's set to that. It doesn't need to be bumped every time there's a release.

Reply:
A that makes sense, thanks